synapse/lib/modelrev.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, core):
             ((0, 2, 15), self.revModel20221212),
             ((0, 2, 16), self.revModel20221220),
             ((0, 2, 17), self.revModel20230209),
+            ((0, 2, 18), self.revModel_0_2_18),
```

Comment:
Do we want to use the date for consistency?

Reply:
I'd like to change the convention here. Dates aren't useful and tend to rot in the PR. I'll create a story to go back through the rest.